package.json

```diff
@@ -2,45 +2,47 @@
   "name": "siwt",
   "version": "0.0.6",
```

Comment:
I guess this should be 0.0.7 but we need to bump to 0.0.8 anyway for the release